src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/__snapshots__/ArtworkContextArtist-tests.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AllArtists renders properly 1`] = `"<view flexdirection=\\"column\\" alignitems=\\"center\\"><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"22px,32px\\" lineheight=\\"28px,38px\\" class=\\"sc-bwzfXH uTxCW\\" mb=\\"2\\" textalign=\\"center\\">Other works by Abbas Kiarostami</text></view>"`;
```

Comment:
did you figure out why this is not rendering the grid?

Reply:
not really, maybe because i stop at `Header`?  do you think i should just switch to a normal test and ignore this snapshot magic?

Reply:
Yeah. And I think the test should make sure that the grid is there and has the expected number of cards in it.